processing/proc_MARA.m

```diff
@@ -0,0 +1,299 @@
+function [goodcomp, info] = proc_MARA(cnt_ica,clab,A)
+% Usage:
+%   [goodcomp, info] = proc_MARA(cnt_ica,clab,A)
+%
+%  IN   cnt_ica     - data structure of independent components 
+%                       cnt_ica.x - [nTimePoint x nComponents]
+%                       cnt.fs    - sampling frequency
+%       mnt         - electrode montage structure
+%       A           - mixing matrix
+% 
+%  OUT  goodcomp    - array containing the numbers of the non-artifactual 
+%                     components
+%       info         - struct containing more information about MARA classification 
+%               info.out   : continuous output of the classifier
+%               info.out_p : out transformed into posterior probablity of being an artifact
+%                            P(artifact | out) (assumption normal distribution)
+%               info.fv_te : feature values for each tested component
+%
+% The classifier is based on the following publiction: 
+%
+% Irene Winkler, Stefan Haufe and Michael Tangermann. Automatic Classification of 
+% Artifactual ICA-Components for Artifact Removal in EEG Signals. 
+% Behavioral and Brain Functions, 7:30, 2011.
+%
+% For more information, see: http://www.user.tu-berlin.de/irene.winkler/artifacts/
+%
+% THIS FUNCTION REQUIRES CLASSIFIER .MAT FILES TO BE INCLUDED IN THE PATH. 
+% The data files  <fv_training_MARA.mat>  and  <inv_matrix_icbm152.mat> are
+% needed. These can be obtained from 
+% http://www.user.tu-berlin.de/irene.winkler/artifacts/MARAtrdata.zip
+
+%%%%%%%%%%%%%%%%%%%%
+%%  Calculate features 
+%%%%%%%%%%%%%%%%%%%%
+load('fv_training_MARA');

```

Comment:
try
    loadData = load(fullfile(BTB.Dir, 'external', 'mara', 'fv_training_MARA.mat'))
    fv_tr = loadData.fv_tr
catch
    error(sprintf('could not load dependencies of MARA. \nMake sure that you have have successfully downloaded the requested files with \n\n> bbci_import_dependencies(''MARA'') '))
end